internal/driver/commands.go

```diff
@@ -337,21 +337,19 @@ func listHelp(c string, redirect bool) string {
 
 // browsers returns a list of commands to attempt for web visualization.
 func browsers() []string {
```

Comment:
Could we match the behavior of Go's cmd/internal/browser/Commands function exactly? Only adding xdg-open when DISPLAY is set is I think reasonable. Not sure why Go's code doesn't use sensible-browser command which is Debian's way to open a default browser. In reality using just xdg-open should be enough, maybe that's why. So how about doing the same, dropping sensible-browser from the list? @hyangah FYI.

Reply:
I don't know why sensible-browser is missing. I guess it's just because that's new in pprof code as well and no one in go side bothered to add it - according to the initial commit of cmd/internal/browser 
https://go-review.googlesource.com/c/go/+/29160/2/src/cmd/internal/pprof/commands/commands.go

We can update cmd/internal/browser to match behaviors.

Reply:
@nolanmar511 Let's probably leave sensible-browser in, after xdg-open in the list. Only xdg-open should be guarded by DISPLAY presence perhaps (since sensible-browser can probably resolve to a terminal browser like elinks). Though frankly I am not sure opening any kind of browser in a non-GUI environment is that useful. I'd probably expect that if DISPLAY is not set, the tool would just print out the URL the user should open by copying and pasting to their browser (think an SSH terminal).

Reply:
Thanks. I assume the code now is line-wise equivalent to the cmd/internal/browser one except `cmds = append(cmds, "sensible-browser")` line. Please merge the PR once CI is green.